relay-server/src/actors/processor.rs

```diff
@@ -1267,6 +1268,95 @@ impl EnvelopeProcessorService {
         Ok(())
     }
 
+    fn process_replays_combine_items(
+        &self,
+        state: &mut ProcessEnvelopeState,
+    ) -> Result<(), ProcessingError> {
+        let project_state = &state.project_state;
+        let combined_envelope_items =
+            project_state.has_feature(Feature::SessionReplayCombinedEnvelopeItems);
```

Comment:
nit: Making this a [guard](https://en.wikipedia.org/wiki/Guard_(computer_science)#Flatter_code_with_less_nesting) might improve readability

```suggestion
        if !project_state.has_feature(Feature::SessionReplayCombinedEnvelopeItems) { return }
```